main.go

```diff
@@ -118,6 +122,18 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *streamKey == "" {
```

Comment:
One downside of that approach is that it's now impossible to intentionally set the stream key to an empty string. We could check if the flag has actually been passed by the user instead: https://stackoverflow.com/questions/35809252/check-if-flag-was-provided-in-go

Reply:
So are you thinking doing something like: 
`./prism -key="" rtmp://something/live/theirkey`

Letting them intentionally set to empty? If so, could also use `-no-key` for more explicit option not to generate key

Reply:
I commited proposed with the readme changes describing it.  :crossed_fingers: